CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
Secondly I just thought about (IMHO) a nice idea how to mark the next release, that yet unreleased. See the below suggestion.

```suggestion
## v2.4.1

### Unreleased
	
```

WDYT? (obviously we can tweak that)

Reply:
I think this might be a bit confusing if someone were to look at the branch and see the version in the changelog without an accompanying release tag. I think that's the reason we don't do this in main either.